SourceryPlugin/Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 
 let package = Package(
     name: "SourceryPlugin",
-    platforms: [.iOS(.v15), .macOS(.v12)],
+    platforms: [.iOS(.v16), .macOS(.v12)],
```

Comment:
To the best of my knowledge and after a little research it is not possible to import anything into these package manifests. However we could use code generation to create some reusable code in these manifest files such as minimum version. I think it would only be worth it if we find there is more code being reused, otherwise the abstraction could complicate things.